src/ape/contracts/base.py

```diff
@@ -939,7 +939,6 @@ def __getattr__(self, attr_name: str) -> Any:
             Any: The return value from the contract call, or a transaction receipt.
         """
 
-        handler: Union[ContractEvent, ContractCallHandler, ContractTransactionHandler]
```

Comment:
Why remove this in this PR?

Reply:
made to separate PR: https://github.com/ApeWorX/ape/pull/1347
can rebase after that merges

Reply:
was part of another PR that merged anyway, but flake8 upgrade pr open nonetheless. resolving convo!